core/modules/filters/each.js

```diff
@@ -16,22 +17,31 @@ Filter operator that selects one tiddler for each unique value of the specified
 Export our filter function
 */
 exports.each = function(source,operator,options) {
-	var results = [],
-		values = {};
-	source(function(tiddler,title) {
-		if(tiddler) {
-			var value;
-			if((operator.operand === "") || (operator.operand === "title")) {
-				value = title;
-			} else {
-				value = tiddler.getFieldString(operator.operand);
+	var results =[] ,
+		value,values = {},
+		field = operator.operand || "title";
+	if("list" !== operator.suffix) {
+		source(function(tiddler,title) {
+			if(tiddler) {
+				value = "title" === field ? title : tiddler.getFieldString(field);
```

Comment:
This would be more consistent as `value = field === "title" ? title : tiddler.getFieldString(field)`. In a case like this I think it's often less confusing to add some braces too:  `value = (field === "title") ? title : tiddler.getFieldString(field)`.